siliconcompiler/tools/openroad/scripts/sc_apr.tcl

```diff
@@ -405,7 +405,7 @@ if { $sc_task == "show" || $sc_task == "screenshot" } {
 
   report_units_metric
 
-  utl::set_metrics_stage "sc__prestep__{}"
+  utl::push_metrics_stage "sc__prestep__{}"
```

Comment:
push_metrics_stage  preserves the previous stack of metrics, set_metrics_stage just clears it.
This will be useful once we can start collecting finer-grain metrics in the actual scripts as well. Right now it's a NOP.